test/integration/pod_e2e_test.go

```diff
@@ -113,12 +114,13 @@ var _ = Describe(fmt.Sprintf("%s,%s: NetworkPolicy On Pods", focusAws, focusAzur
 		if len(kind) > 0 {
 			anpParams.To = &k8stemplates.ToFromParameters{
 				Entity: &k8stemplates.EntitySelectorParameters{
-					Kind:              strings.ToLower(kind),
-					VPC:               strings.ToLower(vpc),
-					CloudInstanceName: strings.ToLower(instanceName),
+					Kind:              config.ExternalEntityLabelKeyKind + ": " + strings.ToLower(kind),
```

Comment:
why skip the len check on this.. On the template side you check for .Entity.Kind, which will be always set now with this approach..